### PR TITLE
Support google jwt auth

### PR DESCRIPTION
**Pull request by sayanarijit**

### CODE REVIEW

**Review by ujjwalmishra09 — APPROVED** [review]

**Comment by ujjwalmishra09** [comment]
@shon  please have a look.

File: apphelpers/socialauth/goog.py [hunk 1]
```diff
@@ -11,7 +13,15 @@
     os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "True"
 
 
-def fetch_info(access_token):
-    session = OAuth2Session(token={"access_token": access_token})
-    userinfo = session.get("https://www.googleapis.com/oauth2/v1/userinfo").json()
+def fetch_info(access_token, jwt=False):
```
Comment (gauravr) [inline apphelpers/socialauth/goog.py:16]: @sayanarijit we can create a separate method
```fetch_info_using_jwt(token):```

Comment (sayanarijit) [reply apphelpers/socialauth/goog.py:16]: Need to create prs for all repos again.

Comment (sayanarijit) [reply apphelpers/socialauth/goog.py]: Updated.

**Review by gauravr — APPROVED** [review]

**Pull request merged**